src/main/java/uk/gov/hmcts/reform/prl/controllers/ManageOrdersController.java

```diff
@@ -162,56 +161,85 @@ public CallbackResponse fetchOrderDetails(
         @ApiResponse(responseCode = "200", description = "Order details are fetched"),
         @ApiResponse(responseCode = "400", description = "Bad Request", content = @Content)})
     @SecurityRequirement(name = "Bearer Authentication")
-    public CallbackResponse prepopulateFL401CaseDetails(
+    public AboutToStartOrSubmitCallbackResponse prepopulateFL401CaseDetails(
         @RequestHeader(org.springframework.http.HttpHeaders.AUTHORIZATION) @Parameter(hidden = true) String authorisation,
         @RequestBody CallbackRequest callbackRequest
     ) {
+        Map<String, Object> caseDataUpdated = callbackRequest.getCaseDetails().getData();
         CaseData caseData = objectMapper.convertValue(
             callbackRequest.getCaseDetails().getData(),
             CaseData.class
         );
-        caseData = caseData.toBuilder()
+        /* log.info("Print selectedC21Order before  set:: {}", caseData.getCreateSelectOrderOptions());
+        log.info("Print manageOrdersOptions before set:: {}", caseData.getManageOrdersOptions());
+        log.info("Print selectedOrder before set:: {}", caseData.getSelectedOrder()); */
+
+        caseDataUpdated.put("selectedC21Order", (null != caseData.getManageOrders()
+            && caseData.getManageOrdersOptions() == ManageOrdersOptionsEnum.createAnOrder)
+            ? caseData.getCreateSelectOrderOptions().getDisplayedValue() : " ");
+        caseDataUpdated.put("manageOrdersOptions", caseData.getManageOrdersOptions());
+
+        /* caseData = caseData.toBuilder()
             .selectedC21Order((null != caseData.getManageOrders()
```

Comment:
can you please remove commented code, if not needed?

Reply:
commented lined removed